openstack/ironic/values.yaml

```diff
@@ -33,6 +33,12 @@ debug: "True"
 
 kos_conductor: true
 
+vault:
+  region: qa-de-1
```

Comment:
Could you remove that default? It is only valid in one region, and if left out will result in the wrong credentials being used.